CONTRIBUTING.md

```diff
@@ -484,6 +484,8 @@ The `dist/` folder within each module contains the packaged up language artifact
 
 ### Quick Iteration
 
+Make sure you have configured these [aliases](#useful-aliases) to your shell profile.
```

Comment:
These should be a recommendation not a requirement. i.e. "We recommend that..."

Reply:
for the quick iteration section, all of the content that follows is contingent on these aliases being configured.

Alternatively, I could change the  content to use npm / yarn in the examples. And this can stay in the useful aliases section.